data/data-pipeline/Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV PYTHONFAULTHANDLER=1 \
   POETRY_VERSION=1.1.12
 
 WORKDIR /data-pipeline
-COPY poetry.lock /data-pipeline/
+COPY pyproject.toml poetry.lock /data-pipeline/
```

Comment:
You don't need the `poetry.lock` here -- it will be generated by the `poetry install` below.

Reply:
I thought we wanted to install the locked version of the dependencies.

https://python-poetry.org/docs/cli/#install

> If there is a poetry.lock file in the current directory, it will use the exact versions from there instead of resolving them. 

If that's not true we can remove it, I just thought this was the intention.